docs-preview/media/alert-styles.css

```diff
@@ -1,16 +1,74 @@
 blockquote.NOTE {
-	font: 14px/20px italic Times, serif;
-	padding: 8px;
-	background-color: #bcc0fa;
-	border-top: 1px solid #bcc0fa;
-	border-bottom: 1px solid #bcc0fa;
-	margin: 5px;
-	background-position: top left;
-	background-repeat: no-repeat;
-	text-indent: 23px;
-	}
-	blockquote.NOTE span {
-		  display: block;
-		  background-repeat: no-repeat;
-		  background-position: bottom right;
-		}
+  font: 14px/20px italic Times, serif;
```

Comment:
It looks like the only part of these blockquote styles that are not shared are the background attributes?

Maybe you can make a common blockquote.alert class with the shared values, and then keep the other items separate?